projects/demo/src/modules/charts/line-days-chart/examples/2/index.html

```diff
@@ -50,7 +49,7 @@
             class="chart"
             [height]="200"
             [width]="getWidth(range)"
-            [value]="chart | tuiFilter : filter : range | tuiMapper : toNumbers : range"
+            [value]="chart | tuiFilter: filter:range | tuiMapper: toNumbers:range"
```

Comment:
It's not critical but could you check if there's an issue on prettier parser to add those spaces? I feel it's much better to read with spaces. Create one if there's no such issue, please.

Reply:
https://github.com/prettier/prettier/issues/13058